addons/website_sale/views/templates.xml

```diff
@@ -475,7 +475,7 @@
                                 <span t-field="product.availability_warning" />
                             </div>
                             <hr t-if="product.description_sale" />
-                            <div>
+                            <div contenteditable="false">
                                 <p t-field="product.description_sale" class="text-muted" />
                             </div>
```

Comment:
yes you are right but in spec "add image" and "remove all image" will on the top and then "background" and then "mode/styling/...." so i did that